openadapt/window/_windows.py

```diff
@@ -51,9 +50,8 @@ def get_active_window_state() -> dict:
     return state
 
 
-def get_active_window_meta(active_window) -> dict:
-    """
-    Get the meta information of the active window.
+def get_active_window_meta(active_window: Any) -> dict:
```

Comment:
should be pywinauto.application.Application

Reply:
addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/6159a69dd7340dd520bd020a57101dfd9d444c9f